### ISSUE TITLE
Advanced User Access Controls

### ISSUE THREAD

**Issue by ispjstfu** [beaverbuilder/feature-requests#62]
It would be great to be able to manually configure what kind of editing functionality is available in the Builder when giving access to different roles.

Currently there is very basic frontend editing, "Builder Access", and "Unrestricted", which is way too much for most clients.

It doesn't have to be too granular, but a little bit more control to at least decide if a client can edit but not move a column or row, if the Style nd Advanced tab should be available, etc.

**Comment by gbissland** [comment]
I think this needs improvement as well. I thought I had submitted this already but think I just talked about it in slack.

We have a lot of cases where we are using column backgrounds and column settings are not currently available to edit unless we make our clients unrestricted.

The User Access Extended Plugin is not a workable solution due to the amount of time taken to set this up for each site/page area.

Current restricted access from BB Kb.

<img width="664" alt="Image 2020-09-23 at 6 06 05 AM" src="https://user-images.githubusercontent.com/5170216/93920357-3326ec80-fd63-11ea-94fe-5804a5118104.png">

**Comment by Bobolinos** [comment]
Great idea +1

**Comment by JessieBee3** [comment]
I agree. It would be nice to be able remove the access to Beaver Builder even on other administrative level accounts. Like only current user (which would be me) can access Beaver Builder editing.
